demo/index.html

```diff
@@ -61,14 +61,10 @@
       <h1>The raml-request-parameters-editor element</h1>
       <div main>
         <template is="dom-if" if="[[isMethod]]" restamp="true">
-          <raml-request-parameters-editor
-            query-parameters="[[selected.queryParameters]]"
-            uri-parameters="[[selected.allUriParameters]]"
-            url="[[selected.absoluteUri]]"
-            value="{{url}}"></raml-request-parameters-editor>
+          <raml-request-parameters-editor query-parameters="[[selected.queryParameters]]" uri-parameters="[[selected.allUriParameters]]" url="[[selected.absoluteUri]]" value="{{url}}"></raml-request-parameters-editor>
```

Comment:
We can use `<raml-request-parameters-editor ...params />` here due to we don't have children for the component. Besides that, all parameters better to have in new lines.

Try using `npm run lint` to see issues.